include/pipettes/core/pipette_info.hpp

```diff
@@ -15,29 +15,22 @@ namespace pipette_info {
 using namespace can::ids;
 using namespace can::messages;
 
-enum class PipetteName {
-    P1000_SINGLE = 0,
-    P1000_MULTI = 1,
-    P1000_96 = 2,
-    P1000_384 = 3,
-};
-
-struct PipetteInfo {
-    PipetteName name;
-    uint16_t model;
-    eeprom::serial_number::SerialNumberType serial;
-};
-
-// This is currently implemented in pipette-type-specific source files in core
-// e.g. pipettes/core/pipette_type_single.cpp but could probably also be parsed
-// from the serial number like the model below
-PipetteName get_name();
-
-// These defines are used to help parse pipette serials
-// A full Serial number looks like P1KSV20201907243 and contains the name, model
-// and individual data code [name]P1K [model]SV [data_code] 20201907243
+// These defines are u    sed to help parse pipette serials, which are stored in
+// a structured binary +     ascii format:
+// +-----+--------+------+
+// |0x00 |name    |binary|
+// |0x01 |        |      |
+// +-----+--------+------+
+// |0x02 |model   |binary|and individual data code [name]P1K [model]SV
+// [data_code] 20201907243 |0x03 |        |      |
```

Comment:
The table formatting is a little hard to read here.

Reply:
oh this got screwed up i think by auto formatting, dang